src/dev/license_checker/config.ts

```diff
@@ -52,6 +52,7 @@ export const LICENSE_ALLOWED = [
   'FreeBSD',
   'ISC',
   'ISC*',
+  'LGPL-3.0+',
```

Comment:
I'm thinking we should probably limit the allow list to only openpgpjs under LGPL-3.0+ instead of globally approving.  I don't think this is supported ATM, taking a look..

Reply:
@jbudz I'm happy to have a go at amending `assertLicensesValid`[here](https://github.com/hop-dev/kibana/blob/f3d69b8197e9119a2fb1d9ee3e8c8af880fd9c9f/src/dev/license_checker/valid.ts#L27) so that is takes a map of packages which are allowed certain licenses, this would be the same structure as `LICENSE_OVERRIDES` [here](https://github.com/hop-dev/kibana/blob/cc00aec8b9337c8a63174f4c0ccc8925a71a1bce/src/dev/license_checker/config.ts#L77)

Reply:
@hop-dev that would be awesome if you have time!

Reply:
@jbudz I've raised a seperate PR for this https://github.com/elastic/kibana/pull/135336